app/src/main/java/com/sopt/dive/feature/login/LoginScreen.kt

```diff
@@ -88,51 +88,38 @@ fun LoginScreen(
         }
 
         OutlinedTextField(
-            value = idText,
-            onValueChange = { idText = it },
+            value = uiState.userId,
+            onValueChange = { viewModel.updateUserId(it) },
```

Comment:
```suggestion
            onValueChange = viewModel::updateUserId,
```
식으로도 사용해줄 수 있답니다ㅎㅎ
해당 방식을 메소드 레퍼런스라고 부르는데, 함수 참조를 직접 사용하므로 추가 람다 객체를 생성하지 않고, 리컴포지션 시 객체 재생성이 없어 성능에 유리하다는 장점이 있습니다.
Compose는 리컴포지션이 빈번하게 발생하므로, 람다 표현식에 의한 객체 생성이 성능에 영향을 미칠 수도 있어요!